theme-contract/src/main/java/com/orange/ouds/theme/OudsColors.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.theme
+
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.graphics.Color
+import com.orange.ouds.theme.tokens.semantic.OudsColorToken
+
+class OudsColors(
+    val primary: Color,
+    val onPrimary: Color,
+    val primaryContainer: Color,
+    val onPrimaryContainer: Color,
+    val inversePrimary: Color,
+    val secondary: Color,
+    val onSecondary: Color,
+    val secondaryContainer: Color,
+    val onSecondaryContainer: Color,
+    val tertiary: Color,
+    val onTertiary: Color,
+    val tertiaryContainer: Color,
+    val onTertiaryContainer: Color,
+    val background: Color,
+    val onBackground: Color,
+    val surface: Color,
+    val onSurface: Color,
+    val surfaceVariant: Color,
+    val onSurfaceVariant: Color,
+    val surfaceTint: Color,
+    val inverseSurface: Color,
+    val inverseOnSurface: Color,
+    val error: Color,
+    val onError: Color,
+    val errorContainer: Color,
+    val onErrorContainer: Color,
+    val outline: Color,
+    val outlineVariant: Color,
+    val scrim: Color,
+    val positive: Color,
+    val onPositive: Color,
+    val negative: Color,
+    val onNegative: Color,
+    val info: Color,
+    val alert: Color,
```

Comment:
For the moment I didn't implement Colors cause this will be the object of the Colors tokens task. So I think we can wait to this task we have to finalize with Maxime before implementing it. This will move.